frontend/src/component/admin/apiToken/CreateApiToken/CreateApiToken.tsx

```diff
@@ -24,9 +24,11 @@ import {
 } from '@server/types/permissions';
 
 const pageTitle = 'Create API token';
+
```

Comment:
not sure what happened here :)

Reply:
I think I run code formatting on another file, sorry 😅